_rules/element-lang-valid-de46e4.md

```diff
@@ -59,24 +60,28 @@ There are differences in how assistive technologies handle unknown and invalid l
 
 #### Passed Example 1
 
-The `lang` attribute has a value that is not empty ("") and has a valid language tag.
+The `lang` attribute has a value that is not empty (`""`) and has a valid language tag.
 
 ```html
 <html>
 	<body>
-		<article lang="en"></article>
+		<article lang="en">
+			They wandered into a strange Tiki bar on the edge of the small beach town.
+		</article>
 	</body>
 </html>
 ```
 
 #### Passed Example 2
 
-The `lang` attribute has a value that is not empty ("") and has a valid language tag. The region section in the value is ignored by the rule.
+The `lang` attribute has a value that is not empty (`""`) and has a valid language tag. The region section in the value is ignored by the rule.
 
 ```html
 <html>
 	<body>
-		<blockquote lang="fr-CH"></blockquote>
+		<blockquote lang="fr-CH">
+			They wandered into a strange Tiki bar on the edge of the small beach town.
```

Comment:
This needs to be French, otherwise it doesn't pass the SC.

Reply:
```suggestion
			Les livres sont des amis froids et sûrs.
```

A little Victor Hugo perhaps?